ted2zim/utils.py

```diff
@@ -1,41 +1,23 @@
 #!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# vim: ai ts=4 sts=4 et sw=4 nu
 
-from urlparse import urljoin
+from urllib.parse import urljoin
 from time import sleep
 import requests
 
-"""
-Utils class for the scraper.
-"""
-__title__ = "utils"
-__author__ = "Rashiq Ahmad"
-__license__ = "GPLv3"
-
 
 def create_absolute_link(base, rel_url):
```

Comment:
I think is is not needed. It's used once and does very little

Reply:
Yup. You're correct. Removed this.